spark/src/test/java/org/apache/iceberg/spark/source/TestSparkTableUtil.java

```diff
@@ -162,4 +163,33 @@ public void testImportUnpartitionedTable() throws Exception {
     long count = spark.read().format("iceberg").load(location.toString()).count();
     Assert.assertEquals("three values ", 3, count);
   }
+
+  @Test
+  public void testImportAsHiveTable() throws Exception {
+    spark.table(qualifiedTableName).write().mode("overwrite").format("parquet")
+            .saveAsTable("unpartitioned_table");
+    TableIdentifier source = spark.sessionState().sqlParser()
+            .parseTableIdentifier("unpartitioned_table");
+
+    HiveCatalog catalog = new HiveCatalog(spark.sparkContext().hadoopConfiguration());
```

Comment:
We should always close `catalog` not to leak resources.

Reply:
Got it. Thanks.

Reply:
I 'd like to change to use the static catalog from `HiveMetastoreTest` to avoid forgetting closing resource.